prosody/rootfs/defaults/conf.d/jitsi-meet.cfg.lua

```diff
@@ -2,6 +2,17 @@ admins = { "{{ .Env.JICOFO_AUTH_USER }}@{{ .Env.XMPP_AUTH_DOMAIN }}" }
 plugin_paths = { "/prosody-plugins/", "/prosody-plugins-custom" }
 http_default_host = "{{ .Env.XMPP_DOMAIN }}"
 
+{{ if .Env.TURN_ENABLE | default "0" | toBool }}
+turncredentials_secret = "{{ .Env.TURN_SECRET | default "keepthissecret" }}";
+turncredentials = {
+  { type = "{{ .Env.TURN_TYPE | default "turns" }}",
+    host = "{{ .Env.TURN_HOST | default .Env.DOCKER_HOST_ADDRESS }}",
```

Comment:
Shouldn't the default be TURN_PUBLIC_IP ?

Reply:
prosody doesn't know about TURN_PUBLIC_IP in case TURN_PUBLIC_IP is not set from env file (got dynamically)

Reply:
We can detect it in the init file and re-export it, perhaps?

Reply:
I'll try to explain my view on it. `TURN_HOST` is an announced FQDN for reach cluster of turn. It has use inside prosody for provide XEP-0215.
`TURN_PUBLIC_IP` is a exactly external IP address of turn instance. 
When we use `TURN_HOST`="turn.example.com", conditionally, `A` record can be `turn.example.com TURN_PUBLIC_IP1, TURN_PUBLIC_IP2, TURN_PUBLIC_IP3` and client (that got `TURN_HOST` via XEP-0215) goes to this scope.

Reply:
On the other hand, if it's just a single setup. TURN_HOST can be DOCKER_HOST_ADDRESS (if not set) and TURN_PUBLIC_IP is dynamically set.

Reply:
I'm not completely sure of the latest comment.. I guess, we should remove default set for "TURN_HOST" and make it force needed, because if TURN_PUBLIC_IP and DOCKER_HOST_ADDRESS is different, setup will broken. WDYT?

Reply:
Not sure I follow you here. I do understand that you could set a hostname. But if you don't, the IP that should be there is the publicc IP of the TURN server, right?

On hindsight DOCKER_HOST_ADDRESS was not a great name. I'll probably rename it to JVB_PUBLIC_IP in the future, which is a better name.

So, to me, it makes sense that the default is the TURN public IP, and not anything else.

Reply:
I suggest to use only `TURN_HOST` for spreading hostname via XEP-0215. In this case we are suppose that:
a) the turn instance only one, and we exactly know how to reach it from outside.
b) the turn instance is multiple, but behind proxy. so, proxy knows how to reach each of them.

The `TURN_HOST` is mandatory variable, otherwise do not enable turn functions.

WDYT?